src/main/webapp/resources/js/pages/analyses/analysis-outputs-table.js

```diff
@@ -121,7 +97,7 @@ async function downloadSelected($dlButton, api) {
     const outputs = selectedNodes.map(node => node.data);
     const { data, error } = await prepareAnalysisOutputsDownload(outputs);
     if (error) {
-      console.error(I18N["analysis.batch-download.ajax.error"], error);
+      console.error(i18n("analysis.batch-download.ajax.error"), error);
```

Comment:
Everything looks ok, but I'm getting a 404 error in the console on this page:
```
GET http://localhost:8080/dist/css/analyses-table.bundle.css net::ERR_ABORTED 404 (Not Found)
```

Reply:
This looks like an old link.  Removed in df2230cf2172f62d7d4cbc5e7d09d7a6498778eb